src/campaigns/youMeUsWe/GridList/GridList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import Link from 'gatsby-link'
```

Comment:
To mute the LGTM alert, we need to add a comment next to line 2
`import Link from 'gatsby-link' // lgtm [js/unused-local-variable]`